src/App.vue

```diff
@@ -17,18 +17,18 @@
 					@category-selected="onSelectCategory"
 					@note-deleted="onNoteDeleted"
 				/>
-				<NcAppNavigationItem
-					:title="t('notes', 'Help')"
-					:pinned="true"
-					@click.prevent="openHelp"
-				>
-					<InfoIcon slot="icon" :size="20" />
-				</NcAppNavigationItem>
-				<AppHelp :open.sync="helpVisible" />
 			</template>
 
 			<template #footer>
-				<AppSettings v-if="!loading.notes && error !== true" @reload="reloadNotes" />
+				<ul class="app-navigation-entry__settings">
+					<NcAppNavigationItem
+						:title="t('notes', 'Notes settings')"
```

Comment:
Not blocking as we can follow up.

@nimishavijay What would be your take on the wording here? Should we maybe call this "Notes settings / help" since the content is a bit more than just settings? See screenshots from before for a full overview.